### ISSUE TITLE
add old projects

### ISSUE THREAD

**Issue by eMBee** [fossasia/labs.fossasia.org#15]
to show off what has been achieved in the past, old projects should be added and marked as closed.

**Comment by todipratik** [comment]
isn't it already done ?

**Comment by eMBee** [comment]
Excerpts from Pratik Todi's message of 2015-02-17 05:11:52 +0100:
> isn't it already done ?

there are many projects that were done during the past years in GSoC:
http://www.google-melange.com/gsoc/org2/google/gsoc2014/fossasia (9 projects)
http://www.google-melange.com/gsoc/org2/google/gsoc2012/fossasia (6 projects)
http://www.google-melange.com/gsoc/org2/google/gsoc2011/fossasia (3 projects)

there may be other projects elsewhere, but it would be nice to have at least
these 18 listed above added.

ideally, with title, description, mentors, student, link to result, link to
gsoc page, and, if it can be found, a link to the current location of the
project, like project homepage or git repository

for the date, just match the year and pick something for month and day.

**Comment by todipratik** [comment]
#25

**Comment by eMBee** [comment]
thank you, this is complete except for a typo in the status: closed vs close. it should be "closed"

**Comment by ty221** [comment]
@eMBee Yeah, that could be considered as a typo, however to minimalize work needed to fix it in all projects I just made this PR - https://github.com/fossasia/labs.fossasia.org/pull/27

Now it gonna work for both "close" and "closed"

**Comment by eMBee** [comment]
it probably does not matter much in this case, but doing things well is a good habit. so i prefer to see the files fixed

**Comment by ty221** [comment]
@eMBee Ok I get your point. However I think you can merge my PR to keep site working also when there are issues of this kind.

**Comment by ty221** [comment]
After #28 is closed we can close this one too.

**Comment by eMBee** [comment]
Excerpts from Tymon Radzik's message of 2015-02-19 20:32:18 +0100:
> After #28 is closed we can close this one too.

yes, by adding "closes #15" in your commit message

**Comment by ty221** [comment]
@eMBee Oh right,  however I didn't add that to my commit desc, because I wasn't sure you don't want to add there anything else. It will be no problem to close this manually
